e2e-tests/cypress/e2e/Forbidden-login.cy.ts

```diff
@@ -0,0 +1,39 @@
+import { Login } from '../support/page-objects/Login';
+import { Loader } from '../support/page-objects/Loader';
+import { Settings } from '../support/page-objects/Settings';
+
+describe('Forbidden login test', () => {
+  before(() => {
+    Settings.setSettingsData('defaultSettings.yaml');
+  });
+
+  beforeEach(() => {
+    cy.clearCookies();
+    cy.clearLocalStorage();
+    cy.visit('/');
+  });
+
+  it('should show an error message when using incorrect credentials', () => {
```

Comment:
Is there a value in keeping it in a separate test?, IMO we should mix it into a single test, with `cy.log('should show an error message when using incorrect credentials')` 

Ref: https://docs.cypress.io/app/core-concepts/best-practices#Creating-Tiny-Tests-With-A-Single-Assertion

Reply:
I agree, but notice that every our test suite starts with `Login.initialization();` in the `beforeEach`. So, it requires some refactoring to do it like that.

Reply:
I'm not sure if I get it, from my perspective

We have a test suite that shows an error message when using incorrect credentials.

and the next, which

shows an error message when using incorrect credentials and then lets the user log in

So the second suite does exactly the same as the first one, or am missing something 🤔

Reply:
oh, sorry. I misunderstood you. Ok, I removed one test